step-plans/step-plans-core/src/main/java/step/core/execution/ExecutionEngineRunner.java

```diff
@@ -108,8 +111,8 @@ protected PlanRunnerResult execute() {
 				
 				ReportNode rootReportNode = executionContext.getReport();
 				executionContext.setCurrentReportNode(rootReportNode);
-				persistReportNode(rootReportNode);

```

Comment:
Why this change?

Reply:
... because it was used exactly once, so I inlined it. :)